R/reportPreview.R

```diff
@@ -1,7 +1,7 @@
 #' UI for 'Report Preview' module
-#' 
+#'
```

Comment:
@imran3004 there are a lot of formatting edits in your PR which is making it hard to go through all the changes. Is this happening automatically on your end?

Reply:
@Jeff-Thompson12 I think that's because I am using Emacs as my editor. Let me use RStudio.

Reply:
I have merged the `dev` branch into this branch and used RStudio to make changes. Can you please confirm if the formatting edits have been reverted?

Reply:
@imran3004 the edits have not been reverted

Reply:
@AARON-CLARK do we care about white space edits? There is the option to hide them when reviewing file changes.